packages/eas-cli/src/run/android/systemRequirements.ts

```diff
@@ -0,0 +1,45 @@
+import spawnAsync from '@expo/spawn-async';
+import chalk from 'chalk';
+import { existsSync } from 'fs-extra';
+
+import { getAaptExecutableAsync } from './aapt';
+import { adbExecutable } from './adb';
+import { emulatorExecutable } from './emulator';
+
+async function whichAsync(executable: string): Promise<string | null> {
```

Comment:
Do not use `which`, you can check if an executable exists by calling it and checking the error.